### PR TITLE
Projectkk2glider/autogen lua docs 2

### LINKED ISSUE #2947: Lua environment status function, Enhancement request.
It would be helpful in finding bugs while developing a Lua script if Lua could determine its environment.
That is, is it operating a PC simulator or on an actual tx?
This would allow an if statement to decide if something needs to have a simulated value set....such as TxVoltage., SWR, RSSI,....

### PR DESCRIPTION
**Pull request by projectkk2glider**
This goes hand in hand with https://github.com/opentx/lua-reference-guide/pull/9

Also closes #2947

### CODE REVIEW

File: radio/src/lua_api.cpp [hunk 1]
```diff
@@ -104,12 +104,72 @@ void hook(lua_State* L, lua_Debug *ar)
   }
 }
 
+#if defined(PCBTARANIS) && defined(REV9E)
+  #define RADIO "taranisx9e"
```
Comment (bsongis) [inline radio/src/lua_api.cpp]: This will have to be removed when the switch to CMake will be completely done!

**Comment by bsongis** [comment]
Thanks!

File: radio/src/lua_api.cpp (hunk 1 above)
Comment (projectkk2glider) [inline radio/src/lua_api.cpp]: I am not happy whit this part of the code also, but I needed to hack up something. Do you have an example of the right solution?

Comment (bsongis) [inline radio/src/lua_api.cpp]: It will easily fixed in a next release, this is already done somewhere else

**Pull request merged**